src/FAQ/FAQPage.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+const FAQPage = (props) => {
+  return (
+    <div className="faq-page">
+      <h2 className="yel">FAQ</h2>
+      <p><span className="pink">Q:</span> Is this a for-profit venture? </p>
+  		<p><span className="blue">A:</span> No, not at the moment. We are a team of open-source contributors implementing <a className="pink" href="https://github.com/ethereum/wiki/wiki/Sharding-FAQ">sharding</a> for Ethereum because we truly believe the protocol desperately needs this ASAP. As a team, we may consider a for profit venture in the future, but we do not intend to profit directly from our work on sharding.</p>
```

Comment:
Can you split this into multiple lines so it conforms to a reasonable column limit? Maybe 100 characters?